pkg/kinds/deprecated_test.go

```diff
@@ -84,3 +86,61 @@ func TestSuggestedApiVersion(t *testing.T) {
 		})
 	}
 }
+
+// func TestRemovedApiVersion(t *testing.T) {
```

Comment:
Any reason this is being left in since its been replaced with the method below? If this is sticking around for something, thought on marking it w/ a `TODO`?

Reply:
Nope, just forgot to delete it.